dataOrganization.md

```diff
@@ -0,0 +1,62 @@
+# Firestore Data Organization
+## Users
+### Collections
+* [Feeders](#feeders)
+* [Notifications](#notifications)
+### Document Fields
+```ts
+{
+  active: boolean,
+  appIdentifier: string,
+  email: string,
+  firstName: string,
+  id: string,
+  lastName: string,
+  lastOnlineTimestamp: timestamp,
+  phoneNumber: string,
+  profilePictureURL: string
+} 
+```
+## Feeders
+### Collections
+* [Pets](#pets)
+### Document Fields
+```ts
+{
+  id: string,
+  modelNumber: string,
+  name: string
+}
+```
+## Notifications
+### Document Fields
+```ts
+{
+  id: string,
+  message: string,
+  time: timestamp
+}
+```
+## Pets
+### Collections
+* [Schedule](#schedule)
+### Document Fields
+```ts
+{
+  RFID: string,
+  age: number,
+  id: string,
+  name: string,
+  type: string,
+  weight: number
+}
+```
+## Schedule
+### Document Fields
+```ts
+{
+  id: string,
```

Comment:
Is this the user id or the pet id or a schedule id?

Reply:
This is the automatically generated document ID that Firebase will assign. I will specify that in a revision.